Owl/app/src/androidTest/java/com/example/owl/ui/fakes/TestImageLoader.kt

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.owl.ui.fakes
+
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.graphics.drawable.Drawable
+import coil.Coil
+import coil.ComponentRegistry
+import coil.ImageLoader
+import coil.decode.DataSource
+import coil.disk.DiskCache
+import coil.memory.MemoryCache
+import coil.request.DefaultRequestOptions
+import coil.request.Disposable
+import coil.request.ImageRequest
+import coil.request.ImageResult
+import coil.request.SuccessResult
+import kotlinx.coroutines.CompletableDeferred
+
+/**
+ * Replaces all remote images with a simple black drawable to make testing faster and hermetic.
+ */
+fun installTestImageLoader() {
```

Comment:
Refactored this class slightly as it used `LocalImageLoader`, which is now deprecated.